src/plonk/mod.rs

```diff
@@ -89,6 +89,9 @@ pub struct PlonkStructure<F: PrimeField> {
 
     /// singla polynomial relation that combines custom gates and lookup relations
     pub(crate) poly: MultiPolynomial<F>,
+
+    pub(crate) custom_gates_lookup_compressed: Expression<F>,
```

Comment:
NBD: this name is okay, but maybe there are better names...

Reply:
Well I've considered it, we don't have a generalized name for this kind of representation, this expr isn't directly used much anywhere, so I think a detailed name is better than a short `compressed` and a rustdoc with an explanation